include/micm/solver/state.hpp

```diff
@@ -139,6 +182,10 @@ namespace micm
     /// @brief Print state (concentrations) at the given time
     /// @param time solving time
     void PrintState(double time);
+
+    /// @brief Default constructor
+    /// Only defined to be used to create default values in types, but a default constructed state is not useable
+    State();
```

Comment:
Would it make more sense to put them in where the `State(...)` constructors are defined?

Reply:
Sure, I can move it back. I had made it private briefly which is why it's here. But that can't work becuase musica does need to make a default value for it to make the MICM object there